index.js

```diff
@@ -1,18 +1,133 @@
+  let guessCount = 0;
+  //computer guess
+  let computerGuess = Math.floor((high - low) / 2) + low;
+  //myAnswer is Y or N
+  let myAnswer = await lib.ask(`Is it ${computerGuess}, yes (y) or no (n)?`);
+  guessCount++
+
+
+  //Cheat detector guessing above
+  function highCheatTest(compGuess, limit) {
+    if (compGuess >= limit) {
+      console.log(`But you said it was lower than ${parseInt(limit + 1)}! I don't play with cheaters, goodbye!`);
+      process.exit();
+    } else {
+      return true
+    }
+  };
+  // Cheat detector guessing below 
+  function lowCheatTest(compGuess, limit) {
+    if (compGuess <= limit) {
+      console.log(`But you said it was higher than ${parseInt(limit - 1)}! I don't play with cheaters, goodbye!`);
+      process.exit();
+    } else {
+      return true
+    }
+  };
+  //wrong guess answered
+  while (myAnswer.toLowerCase() === 'n') {
+    let highLow = await lib.ask('Is it higher (h) or lower (l)?')
+    if (highLow.toLowerCase() === 'h' && highCheatTest(computerGuess, high)) {
+      low = computerGuess + 1;
+    } else if (highLow.toLowerCase() === 'l' && lowCheatTest(computerGuess, low)) {
+      high = computerGuess - 1;
+    } else if (highLow.toLowerCase() !== 'h' && highLow.toLowerCase() !== 'l') {
+      console.log("I'm sorry, what?!?!?")
+    }
+    computerGuess = Math.floor((high - low) / 2) + low;
+    myAnswer = await lib.ask(`Is it ${computerGuess}, yes (y) or no (n)?`);
+    guessCount++
+  }
+  //correct guess answered
+  if (myAnswer.toLowerCase() === 'y') {
+    //console.log using ternary operator for correct tenses
+    console.log(`Yes! I got you number and it only took ${guessCount} guess${guessCount > 1 ? "es" : ""} buddy! \n \n`);
+
+    //begin Human guess game
+    startHumGuessNum();
+  }
+
 }
+//Human guess start game
+async function startHumGuessNum() {
+  //range
+  let high = (process.argv[2] || 100);
+  let low = 1;
+
+  console.log(`Let's play a game where I (computer) think up a number and you (human) try to guess it. \nI am thinking of a number between 1 and ${high} (inclusive).`);
+  let secretNumber = randomNum(low, high);
+  let questionText = "What do you think it could be?\n";
+  let myGuess = await lib.ask(questionText);
+  myGuess = parseInt(myGuess);
+
+  //guess count
+  let guessCount = 0;
+  guessCount++
+
+  //correct guess answered
+  if (myGuess === secretNumber) {
+    myGuess = parseInt(myGuess);
+    win();
+  }
+
+  while (myGuess != secretNumber) {
+    //wrong guess answered and it's too high
+    if (myGuess > secretNumber) {
+      console.log(`Nope! ${myGuess} is too high! `);
+      myGuess = await lib.ask("what do you think it could be? ");
+      myGuess = parseInt(myGuess);
+      //wrong guess answered and it's too low        
+    } else {
+      console.log(`Nope! ${myGuess} is too low! `);
+      myGuess = await lib.ask("What do you think it could be? ");
+      myGuess = parseInt(myGuess);
+    }
+    guessCount++
+  }
+  win();
```

Comment:
Having this function be called outside of a conditional that declares the winner might give room for some unintended errors.  Lets say the function above it had a bug that caused it to fall through, then the game would still declare a winner.  One other small note, you currently have a win function for this version of the game, but not the other version.  There is a chance to clean up the code a bit here, and make the win function handle either game.